.travis.yml

```diff
@@ -3,23 +3,15 @@ language: python
 matrix:
   include:
     - python: 2.7
-      env: TOXENV=py27
     - python: pypy
-      env: TOXENV=pypy
     - python: 3.3
-      env: TOXENV=py33
     - python: 3.4
-      env: TOXENV=py34
     - python: 3.5
-      env: TOXENV=py35
     - python: 3.6
-      env: TOXENV=py36
-    - env: TOKENV=flake8
-install: pip install tox
-script: tox
+install: pip install --allow-all-external -e .
+script: nosetests
```

Comment:
I don't really envy you right now... ;)

(FWIW maybe we should think about using [travis-tox](https://github.com/ryanhiebert/tox-travis))?